src/gui/wizard/selftest_fans_axis.cpp

```diff
@@ -1,12 +1,103 @@
-// selftest_fans_axis.c
+// selftest_fans_axis.cpp
+#include "selftest_fans_axis.hpp"
+#include "i18n.h"
+#include "gui.hpp"
+
+StateFncData StateFnc_SELFTEST_FAN0(StateFncData last_run) {
+    static const char *title_txt = N_(
```

Comment:
Do we really need the extra space padding and explicit newlines?
Or is this piece of code subject to change in the further rework?

Reply:
1) I hope we do not. But do not know, did not try. 
It is just moved old code. 
2) Yes

Reply:
1. Sadly we do
tested on
```
    static const char *title_txt = N_(
        "State\n"
        "SELFTEST_X"
        "not implemented");
```
\n is ignored

Reply:
multiline flag?

Reply:
Y

Reply:
did not help